src/ripple/proto/org/xrpl/rpc/v1/README.md

```diff
@@ -0,0 +1,35 @@
+This folder contains the protocol buffer definitions used by the rippled gRPC API.
+The gRPC API attempts to mimic the JSON/Websocket API as much as possible.
+As of April 2020, the gRPC API supports a subset of the full rippled API:
+tx, account_tx, account_info, fee and submit.
+
+### Making Changes
```

Comment:
I'd like at least one blank line after each header. With syntax highlighting, the distinction between headers and paragraphs is easy to see, but in situations without, it can be easy to miss without vertical separation.